Jenkinsfile.txt

```diff
@@ -0,0 +1,114 @@
+pipeline {
+ 
+  agent {
+    kubernetes {
+      label 'hugo-agent'
+      yaml """
+apiVersion: v1
+metadata:
+  labels:
+    run: hugo
+  name: hugo-pod
+spec:
+  containers:
+    - name: jnlp
+      volumeMounts:
+      - mountPath: /home/jenkins/.ssh
+        name: volume-known-hosts
+      env:
+      - name: "HOME"
+        value: "/home/jenkins"
+    - name: hugo
+      image: eclipsecbi/hugo:0.81.0
+      command:
+      - cat
+      tty: true
+  volumes:
+  - configMap:
+      name: known-hosts
+    name: volume-known-hosts
+"""
+    }
+  }
+ 
+  environment {
+    PROJECT_NAME = "swt" // must be all lowercase.
+    PROJECT_BOT_NAME = "SWT Bot" // Capitalize the name
+  }
+ 
+  triggers { pollSCM('H/10 * * * *') 
+ 
+ }
+ 
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '5'))
+    checkoutToSubdirectory('hugo')
+  }
+ 
+  stages {
+    stage('Checkout www repo') {
+      steps {
+        dir('www') {
+            sshagent(['git.eclipse.org-bot-ssh']) {
```

Comment:
This requires an update. The key used here is used by eclipse hosted git repos. not for the github repositories. Please go thought the I-build jenkins file for the correct key